python/ray/serve/controller.py

```diff
@@ -131,6 +132,9 @@ def check_alive(self) -> None:
     def record_autoscaling_metrics(self, data: Dict[str, float], send_timestamp: float):
```

Comment:
I think we should collapse `record_autoscaling_metrics` and `record_handle_metrics` into a single function and pass in the `metrics_store` as an argument.

Reply:
I will have it when we add one more store (for now the func is trivial, so I will keep it as it is). :) I am thinking to have store manager in the following pr.

Reply:
That sounds good!